docs/docs/using-gatsby-image.md

```diff
@@ -2,6 +2,8 @@
 title: Using Gatsby Image to Prevent Image Bloat
 ---
 
+<ImageModel initialLayer="Install" />
```

Comment:
In other instances where the (now) `<ComponentModel />` is used, there's a one sentence preface to describe its context in the doc. Is there some text that could go above this to describe what this model is outlining for it and the rest of the doc?

Reply:
Added. Not sure how I feel about it though. Open to suggestions!